pkg/integrations/cadvisor/cadvisor_stub.go

```diff
@@ -0,0 +1,86 @@
+//go:build !linux
+// +build !linux
+
+package cadvisor //nolint:golint
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/config"
+)
+
+// DefaultConfig holds the default settings for the cadvisor integration
+var DefaultConfig Config = Config{}
```

Comment:
nit: we might end up having a few of these stubs :) Can you add a helper somewhere to make having them less boilerplate?